src/components/views/Agreement.js

```diff
@@ -98,23 +108,43 @@ export default class Agreement extends React.Component {
     this.setState({ sections });
   }
 
-  getActionMenu = () => (
-    <IfPermission perm="ui-agreements.agreements.edit">
+  getActionMenu = (onToggle) => (
+    <Fragment>
+      <IfPermission perm="ui-agreements.agreements.edit">
+        <FormattedMessage id="ui-agreements.agreements.editAgreement">
+          {ariaLabel => (
+            <Button
+              aria-label={ariaLabel}
+              buttonStyle="dropdownItem"
+              id="clickable-dropdown-edit-agreement"
+              onClick={this.props.handlers.onEdit}
+            >
+              <Icon icon="edit">
+                <FormattedMessage id="ui-agreements.agreements.editAgreement" />
+              </Icon>
+            </Button>
+          )}
+        </FormattedMessage>
+      </IfPermission>
       <FormattedMessage id="ui-agreements.agreements.editAgreement">
```

Comment:
`ariaLabel` will be incorrect here.

Also, we should probably wrap this in an `IfPermission`, no? I'm assuming that only users that can edit agreements should use the clone endpoint?